src/app/shared/mydspace-actions/claimed-task/approve_no_diffusion/claimed-task-actions-approve-no-diffusion.component.html

```diff
@@ -0,0 +1,38 @@
+<button type="button"
+        [className]="'btn btn-success'"
+        ngbTooltip="{{'submission.workflow.tasks.claimed.approve-no-diffusion_help' | translate}}"
+        [disabled]="processing$ | async"
+        (click)="openApproveModal(approveModal)">
+  <span *ngIf="processing$ | async"><i class='fas fa-circle-notch fa-spin'></i> {{'submission.workflow.tasks.generic.processing' | translate}}</span>
+  <span *ngIf="!(processing$ | async)"><i class="fa fa-thumbs-up"></i> {{'submission.workflow.tasks.claimed.approve-no-diffusion' | translate}}</span>
+</button>
+
+<ng-template #approveModal let-c="close" let-d="dismiss">
+  <div class="modal-header">
+    <h4 class="modal-title">{{'submission.workflow.tasks.claimed.approve-no-diffusion.reason.title' | translate}}</h4>
+    <button type="button"
+            class="close"
+            aria-label="Close"
+            (click)="d('Cross click')">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="modal-body">
+    <div class="alert alert-info" role="alert">
+      {{'submission.workflow.tasks.claimed.approve-no-diffusion.reason.info' | translate}}
+    </div>
+    <form (ngSubmit)="submitTask();">
+      <textarea style="width: 100%"
```

Comment:
simply use`class="form-control"`
Using "style" is (very) rarely a good idea ;-)